rust-bors.toml

```diff
@@ -1,2 +1,5 @@
 # 6 hours timeout for CI builds
 timeout = 21600
+
+# Do not allow approving PRs that are waiting for a FCP to finish
+labels_blocking_approval = ["final-comment-period", "proposed-final-comment-period"]
```

Comment:
Do we also want to block on `S-waiting-on-concerns` ?

Reply:
There's also `S-waiting-on-mcp`, `S-blocked`, etc.

Reply:
draft PRs already will not be eligible for r+

Reply:
> Could authors also add a label for "do not merge" experimental PRs to further guard against them being merged?

(There's also `S-experimental`)

Reply:
Would it be possible to also block certain *combinations* of labels or missing labels? For instance, could we block merging a PR that has `needs-fcp` and *doesn't* have both `finished-final-comment-period` and `disposition-merge`?

Reply:
It surely is possible to implement :D But it might be a bit complex to express these rules in a TOML config file. That being said, since new bors will most likely only be used for rust-lang/rust, we could also just hardcode it :man_shrugging: